frontend/packages/console-app/src/components/nodes/NodeStatus.tsx

```diff
@@ -10,7 +10,7 @@ type NodeStatusProps = {
 const NodeStatus: React.FC<NodeStatusProps> = ({ node }) => (
   <>
     <Status status={nodeStatus(node)} />
-    {isNodeUnschedulable(node) && <small className="text-muted">Scheduling Disabled</small>}
+    <SecondaryStatus>{getNodeSecondaryStatus(node)}</SecondaryStatus>
```

Comment:
Hm, maybe we can do

```tsx
<SecondaryStatus status={getNodeSecondaryStatus(node)} />
```

using an explicit `status` prop is better than passing this info as `children` in my opinion.